tools/gohub/cmd/tools/install/install.go

```diff
@@ -42,6 +42,7 @@ func IncludeDirs() []string {
 	return []string{
 		filepath.Join(home, "."+cmd.Name, repo, "tools/protoc/include"),
 		filepath.Join(home, "."+cmd.Name),
+		filepath.Join(home, "."+cmd.Name, "github.com/envoyproxy/protoc-gen-validate"),
```

Comment:
why we doesn't need include proto files inside github.com/mwitkow/go-proto-validators before?

Reply:
remove the include.